### PR TITLE
Fixed __add__ and __radd__ in input class

### PR DESCRIPTION
**Pull request by psav**
{{pytest: -k test_provider_add_with_bad_credentials}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f31e14ede558bc554cb8f2bffa37013d1d306027:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/web_ui/__init__.py [hunk 1]
```diff
@@ -1261,10 +1261,10 @@ def locate(self):
         )
 
     def __add__(self, string):
-        return '//input[@name="{}"]{}'.format(self.name, string)
+        return '//input[@name="{}"]{}'.format(self.locate(), string)
```
Comment (mfalesni) [inline cfme/web_ui/__init__.py]: @psav 
```python
return self.locate() + str(string)
```

locate returns already constructed locator

Comment (psav) [inline cfme/web_ui/__init__.py]: oh yeh duh

**Comment by dajoRH** [comment]
Lint report for commit ab6fe0bbe5c5e8bf8b9e4d32dd0935e71895f816:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**